client/src/main/java/model/Ship.java

```diff
@@ -10,13 +10,13 @@
 
     private static final int START_SIZE = 1;
 
-    private final Mast[] masts;
+    private Mast[] masts;
     private final int length;
     private int buildLength;
```

Comment:
can't be final?

Reply:
No, this is increased during building.

Reply:
ok

Reply:
No buildLength is changing during bulding ship